fortinet_fortigate/plugin.spec.yaml

```
@@ -0,0 +1,749 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: fortinet_fortigate
+title: Fortinet FortiGate
+description: FortiGate Next Generation Firewalls (NGFWs) enable security-driven networking and consolidate industry-leading security capabilities such as intrusion prevention system (IPS), web filtering, secure sockets layer (SSL) inspection, and automated threat protection
+version: 1.0.0
+vendor: joey-test
```

```suggestion
vendor: rapid7
```
regenerate after this change